htdocs/core/class/html.formmail.class.php

```diff
@@ -420,7 +420,7 @@ public function get_form($addfileaction = 'addfile', $removefileaction = 'remove
 					$model_id = $this->param["models_id"];
 				}
 
-				$arraydefaultmessage = $this->getEMailTemplate($this->db, $this->param["models"], $user, $outputlangs, $model_id, 1, '', ($model_id > 0 ? -1 : 1)); // If $model_id is empty, preselect the first one
+				$arraydefaultmessage = $this->getEMailTemplate($this->db, $this->param["models"], $user, $outputlangs, $model_id, 1, '', ($model_id > 0 ? 1 : -1)); // If $model_id is empty, preselect the first one
```

Comment:
I don't understand.
If $model_id is set for a default selection (it means we want a given $model_id selected), then we must use -1 in $defaultfortype because we don't want to filter on default, we want to filter on $model_id.
if $model_id is not set, then we must use 1 to get the default value.

Reply:
@SylvainLegrand 
Maybe, give us more details in email templates parameters (some screenshots).
I tried with only one email template and is not selected by default but I forgot to set it as default.
![image](https://github.com/Dolibarr/dolibarr/assets/45359511/d1604433-d56e-40b7-8095-6e3cc699828e)
Did you try setting the email template as default ?